### PR TITLE
Move to latest toolset

### PR DESCRIPTION
**Pull request by davkean**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/VsTelemetryService.cs [hunk 1]
```diff
@@ -65,17 +65,15 @@ protected virtual void PostEventToSession(TelemetryEvent telemetryEvent)
...
-            // Don't hash PII for internal users since we don't need to.
-            if (TelemetryService.DefaultSession.IsUserMicrosoftInternal)
-            {
-                return value;
-            }
+            //// Don't hash PII for internal users since we don't need to.
+            //if (TelemetryService.DefaultSession.IsUserMicrosoftInternal)
+            //{
+            //    return value;
+            //}
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/VsTelemetryService.cs]: Whats the story here?

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/VsTelemetryService.cs]: Great catch, that we was me repro'ing a bug.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Input/Commands/AbstractAddItemCommandHandlerTests.cs [hunk 2]
```diff
@@ -202,9 +202,8 @@ public async Task TryHandleCommand_FolderAsNodes_ReturnsTrueWhenUserClicksCancel
         internal AbstractAddItemCommandHandler CreateInstance(IPhysicalProjectTree projectTree = null, IUnconfiguredProjectVsServices projectVsServices = null, IProjectTreeProvider provider = null, IVsAddProjectItemDlg addItemDialog = null, string capability = null)
         {
             var configuredProject = ConfiguredProjectFactory.Create(IProjectCapabilitiesScopeFactory.Create(new string[] { capability ?? TestAddItemCommand.Capability }));
-            projectTree = projectTree ?? IPhysicalProjectTreeFactory.Create(provider);
-            projectVsServices = projectVsServices ??
-                IUnconfiguredProjectVsServicesFactory.Implement(
+            projectTree ??= IPhysicalProjectTreeFactory.Create(provider);
+            projectVsServices ??=                 IUnconfiguredProjectVsServicesFactory.Implement(
```
Comment (jmarolf) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Input/Commands/AbstractAddItemCommandHandlerTests.cs]: Spacing looks off here.  Can we file a bug on roslyn to fix this?

**Comment by jmarolf** [comment]
@davkean is this ready to merge?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Input/Commands/AbstractAddItemCommandHandlerTests.cs (hunk 2 above)
Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Input/Commands/AbstractAddItemCommandHandlerTests.cs]: https://github.com/dotnet/roslyn/issues/35870

**Pull request merged**